src/cr/cube/cubepart.py

```diff
@@ -695,60 +610,96 @@ def pairwise_means_indices_alt(self):
-            return None
         try:
-            return self._assembler.pairwise_means_indices(
-                self._alpha_alt, self._only_larger
-            )
+            return self._assembler.pairwise_means_indices_alt
         except ValueError:
             raise ValueError(
                 "`.pairwise_means_indices_alt` is undefined for a cube-result "
                 "without a mean measure"
             )
 
-    def pairwise_significance_p_vals(self, column_idx):
-        """2D ndarray of pairwise-significance p-vals matrices for column idx."""
-        return self._assembler.pairwise_significance_p_vals(column_idx)
+    @lazyproperty
+    def pairwise_significance_p_vals(self):
+        """2D ndarray of p-vals for each pair of compared groups of proportions.
+
+        Considering a 2x3 proportions matrix and selecting the baseline column via the
+        transforms, the p-vals will be a 2x3 matrix where the compared groups are the
+        selected column with each of the other columns.
+
+        E.G. first col selected
+            [
+               [1.0, 0.05, 0.01],
+               [1.0, 0.01, 0.05],
+               [1.0, 0.1, 0.01],
+            ]
+        """
+        return self._assembler.pairwise_significance_p_vals
+
+    @lazyproperty
+    def pairwise_significance_p_vals_scale_means(self):
+        """1D ndarray of p-vals for each pair of compared groups of scale means.
+
+        The p-vals are calculated considering the row dimension numeric values.
+        """
+        return self._assembler.pairwise_p_vals_scale_means
+
+    @lazyproperty
+    def pairwise_significance_t_stats(self):
+        """2D ndarray of t-stats for each pair of compared groups of proportions.
+
+        Considering a 2x3 proportions matrix and selecting the baseline column via the
+        transforms, the t-stats will be a 2x3 matrix where the compared groups are the
+        selected column with each of the other columns.
 
-    def pairwise_significance_t_stats(self, column_idx):
-        """return 2D ndarray of pairwise-significance t-stats for selected column."""
-        return self._assembler.pairwise_significance_t_stats(column_idx)
+        E.G. first col selected
+            [
+               [0.0, -0.004, 0.01],
+               [0.0, 0.041, -0.02],
+               [0.0, 0.001, -0.01],
+            ]
+        """
+        return self._assembler.pairwise_significance_t_stats
 
-    def pairwise_significance_means_p_vals(self, column_idx):
-        """Optional 2D ndarray of means significance p-vals matrices for column idx."""
-        # Significance of means difference is available only is cube contains means.
+    @lazyproperty
+    def pairwise_significance_t_stats_scale_means(self):
+        """1D ndarray of t-stats for each pair of compared groups of scale means.
+
+        The t-stats are calculated considering the row dimension numeric values.
+        """
+        return self._assembler.pairwise_t_stats_scale_means
```

Comment:
If this is 1D, I assume it's a marginal. State in the docstring whether it appears as a summary row or summary column (which implies its length is ncols or nrows respectively). Also add an intuitive explanation for what it is, maybe something like ("Summarizes the ...") so we can make sense of the name. Also indicate what a cell value looks like. Is that a tuple of values like `(0.035, 0.130, ..)` one for each significant value? Do whatever it takes to make it so a user can figure out if this is something they're looking for or not. This docstring is the public documentation for this property.